### ISSUE TITLE
Analysis graphs broken on f8e0190

### ISSUE THREAD

**Issue by jpchow26** [deepfakes/faceswap#868]
Clicking the icon to show graphs on the analysis tab results in just the control palette showing for the popup graph window and following error in output:
```

Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\jpcho\MiniConda3\envs\faceswap\lib\tkinter\__init__.py", line 1705, in __call__
    return self.func(*args)
  File "C:\Users\jpcho\faceswap\lib\gui\display_analysis.py", line 343, in select_item
    self.data_popup(datapoints)
  File "C:\Users\jpcho\faceswap\lib\gui\display_analysis.py", line 366, in data_popup
    datapoints)
  File "C:\Users\jpcho\faceswap\lib\gui\display_analysis.py", line 439, in __init__
    self.build()
  File "C:\Users\jpcho\faceswap\lib\gui\display_analysis.py", line 461, in build
    self.opts_build(optsframe)
  File "C:\Users\jpcho\faceswap\lib\gui\display_analysis.py", line 494, in opts_build
    self.opts_slider(frame)
  File "C:\Users\jpcho\faceswap\lib\gui\display_analysis.py", line 620, in opts_slider
    helptext=self.set_help(item))
TypeError: __init__() got multiple values for argument 'label_width'
```
Screen shot of resulting graph popup
![image](https://user-images.githubusercontent.com/10886704/64709400-2c914400-d484-11e9-9081-b5455119383c.png)

System Info output:

```
============ System Information ============
encoding:            cp1252
git_branch:          master
git_commits:         f8e0190 update opencv-python. da0d303 s3fd-amd fixes. 3ba8c73 de-MultiProcess Training. f083301 De-Multiprocess Convert (#857). 8c21248 Fix misspellings in help messages of arguments (#856)
gpu_cuda:            No global version found. Check Conda packages for Conda Cuda
gpu_cudnn:           No global version found. Check Conda packages for Conda cuDNN
gpu_devices:         GPU_0: GeForce GTX 1060 3GB
gpu_devices_active:  GPU_0
gpu_driver:          436.30
gpu_vram:            GPU_0: 3072MB
os_machine:          AMD64
os_platform:         Windows-10-10.0.18362-SP0
os_release:          10
py_command:          C:\Users\jpcho\faceswap/faceswap.py gui
py_conda_version:    conda 4.7.10
py_implementation:   CPython
py_version:          3.6.8
py_virtual_env:      True
sys_cores:           4
sys_processor:       Intel64 Family 6 Model 42 Stepping 7, GenuineIntel
sys_ram:             Total: 16351MB, Available: 9094MB, Used: 7256MB, Free: 9094MB

=============== Pip Packages ===============
absl-py==0.7.1
astor==0.7.1
certifi==2019.6.16
cloudpickle==1.1.1
cycler==0.10.0
cytoolz==0.9.0.1
dask==1.2.2
decorator==4.4.0
fastcluster==1.1.25
ffmpy==0.2.2
gast==0.2.2
grpcio==1.16.1
h5py==2.9.0
imageio==2.5.0
imageio-ffmpeg==0.3.0
joblib==0.13.2
Keras==2.2.4
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
kiwisolver==1.1.0
Markdown==3.1.1
matplotlib==2.2.2
mkl-fft==1.0.12
mkl-random==1.0.2
mkl-service==2.0.2
mock==3.0.5
networkx==2.3
numpy==1.16.2
nvidia-ml-py3==7.352.1
olefile==0.46
opencv-python==4.1.0.25
pathlib==1.0.1
Pillow==6.1.0
protobuf==3.8.0
psutil==5.6.2
pyparsing==2.4.0
pyreadline==2.1
python-dateutil==2.8.0
pytz==2019.1
PyWavelets==1.0.3
pywin32==223
PyYAML==5.1
scikit-image==0.15.0
scikit-learn==0.21.2
scipy==1.2.1
six==1.12.0
tensorboard==1.13.1
tensorflow==1.13.1
tensorflow-estimator==1.13.0
termcolor==1.1.0
toolz==0.9.0
toposort==1.5
tornado==6.0.2
tqdm==4.32.1
Werkzeug==0.15.4
wincertstore==0.2

============== Conda Packages ==============
# packages in environment at C:\Users\jpcho\MiniConda3\envs\faceswap:
#
# Name                    Version                   Build  Channel
_tflow_select             2.1.0                       gpu  
absl-py                   0.7.1                    py36_0  
astor                     0.7.1                    py36_0  
blas                      1.0                         mkl  
ca-certificates           2019.5.15                     0  
certifi                   2019.6.16                py36_1  
cloudpickle               1.1.1                      py_0  
cudatoolkit               10.0.130                      0  
cudnn                     7.6.0                cuda10.0_0  
cycler                    0.10.0           py36h009560c_0  
cytoolz                   0.9.0.1          py36hfa6e2cd_1  
dask-core                 1.2.2                      py_0  
decorator                 4.4.0                    py36_1  
fastcluster               1.1.25                   pypi_0    pypi
ffmpy                     0.2.2                    pypi_0    pypi
freetype                  2.9.1                ha9979f8_1  
gast                      0.2.2                    py36_0  
grpcio                    1.16.1           py36h351948d_1  
h5py                      2.9.0            py36h5e291fa_0  
hdf5                      1.10.4               h7ebc959_0  
icc_rt                    2019.0.0             h0cc432a_1  
icu                       58.2                 ha66f8fd_1  
imageio                   2.5.0                    py36_0  
imageio-ffmpeg            0.3.0                    pypi_0    pypi
intel-openmp              2019.4                      245  
joblib                    0.13.2                   py36_0  
jpeg                      9c                hfa6e2cd_1001    conda-forge
keras                     2.2.4                         0  
keras-applications        1.0.8                      py_0  
keras-base                2.2.4                    py36_0  
keras-preprocessing       1.1.0                      py_1  
kiwisolver                1.1.0            py36ha925a31_0  
libblas                   3.8.0                     8_mkl    conda-forge
libcblas                  3.8.0                     8_mkl    conda-forge
liblapack                 3.8.0                     8_mkl    conda-forge
liblapacke                3.8.0                     8_mkl    conda-forge
libmklml                  2019.0.3                      0  
libpng                    1.6.37               h2a8f88b_0  
libprotobuf               3.8.0                h7bd577a_0  
libtiff                   4.0.10               hb898794_2  
libwebp                   1.0.2                hfa6e2cd_2    conda-forge
markdown                  3.1.1                    py36_0  
matplotlib                2.2.2            py36had4c4a9_2  
mkl                       2019.4                      245  
mkl-service               2.0.2            py36he774522_0  
mkl_fft                   1.0.12           py36h14836fe_0  
mkl_random                1.0.2            py36h343c172_0  
mock                      3.0.5                    py36_0  
networkx                  2.3                        py_0  
numpy                     1.16.2           py36h19fb1c0_0  
numpy-base                1.16.2           py36hc3f5095_0  
nvidia-ml-py3             7.352.1                  pypi_0    pypi
olefile                   0.46                     py36_0  
opencv                    4.1.0            py36hb4945ee_5    conda-forge
opencv-python             4.1.0.25                 pypi_0    pypi
openssl                   1.1.1c               he774522_1  
pathlib                   1.0.1                    py36_1  
pillow                    6.1.0            py36hdc69c19_0  
pip                       19.1.1                   py36_0  
protobuf                  3.8.0            py36h33f27b4_0  
psutil                    5.6.2            py36he774522_0  
pyparsing                 2.4.0                      py_0  
pyqt                      5.9.2            py36h6538335_2  
pyreadline                2.1                      py36_1  
python                    3.6.8                h9f7ef89_7  
python-dateutil           2.8.0                    py36_0  
pytz                      2019.1                     py_0  
pywavelets                1.0.3            py36h8c2d366_1  
pywin32                   223              py36hfa6e2cd_1  
pyyaml                    5.1              py36he774522_0  
qt                        5.9.7            vc14h73c81de_0  
scikit-image              0.15.0           py36ha925a31_0  
scikit-learn              0.21.2           py36h6288b17_0  
scipy                     1.2.1            py36h29ff71c_0  
setuptools                41.0.1                   py36_0  
sip                       4.19.8           py36h6538335_0  
six                       1.12.0                   py36_0  
sqlite                    3.28.0               he774522_0  
tensorboard               1.13.1           py36h33f27b4_0  
tensorflow                1.13.1          gpu_py36h9006a92_0  
tensorflow-base           1.13.1          gpu_py36h871c8ca_0  
tensorflow-estimator      1.13.0                     py_0  
tensorflow-gpu            1.13.1               h0d30ee6_0  
termcolor                 1.1.0                    py36_1  
tk                        8.6.8                hfa6e2cd_0  
toolz                     0.9.0                    py36_0  
toposort                  1.5                      pypi_0    pypi
tornado                   6.0.2            py36he774522_0  
tqdm                      4.32.1                     py_0  
vc                        14.1                 h0510ff6_4  
vs2015_runtime            14.15.26706          h3a45250_4  
werkzeug                  0.15.4                     py_0  
wheel                     0.33.4                   py36_0  
wincertstore              0.2              py36h7fe50ca_0  
xz                        5.2.4                h2fa13f4_4  
yaml                      0.1.7                hc54c509_2  
zlib                      1.2.11               h62dcd97_3  
zstd                      1.3.7                h508b16e_0  

================= Configs ==================
--------- .faceswap ---------
backend:                  nvidia

--------- convert.ini ---------

[color.color_transfer]
clip:                     True
preserve_paper:           True

[color.manual_balance]
colorspace:               HSV
balance_1:                0.0
balance_2:                0.0
balance_3:                0.0
contrast:                 0.0
brightness:               0.0

[color.match_hist]
threshold:                99.0

[mask.box_blend]
type:                     gaussian
distance:                 11.0
radius:                   5.0
passes:                   1

[mask.mask_blend]
type:                     normalized
radius:                   3.0
passes:                   4
erosion:                  0.0

[scaling.sharpen]
method:                   unsharp_mask
amount:                   150
radius:                   0.3
threshold:                5.0

[writer.ffmpeg]
container:                mp4
codec:                    libx264
crf:                      30
preset:                   ultrafast
tune:                     none
profile:                  auto
level:                    auto

[writer.gif]
fps:                      25
loop:                     0
palettesize:              256
subrectangles:            False

[writer.opencv]
format:                   png
draw_transparent:         False
jpg_quality:              75
png_compress_level:       3

[writer.pillow]
format:                   png
draw_transparent:         False
optimize:                 False
gif_interlace:            True
jpg_quality:              75
png_compress_level:       3
tif_compression:          tiff_deflate

--------- extract.ini ---------

[detect.cv2_dnn]
confidence:               50

[detect.mtcnn]
minsize:                  20
threshold_1:              0.6
threshold_2:              0.7
threshold_3:              0.7
scalefactor:              0.709

[detect.s3fd_amd]
confidence:               50
batch-size:               8

[detect.s3fd]
confidence:               75

--------- gui.ini ---------

[global]
fullscreen:               False
tab:                      extract
options_panel_width:      30
console_panel_height:     20
font:                     default
font_size:                9

--------- train.ini ---------

[global]
coverage:                 68.75
mask_type:                components
mask_blur:                True
icnr_init:                False
conv_aware_init:          False
subpixel_upscaling:       False
reflect_padding:          False
penalized_mask_loss:      True
loss_function:            mae
learning_rate:            5e-05

[model.dfl_h128]
lowmem:                   True

[model.dfl_sae]
input_size:               128
clipnorm:                 True
architecture:             df
autoencoder_dims:         0
encoder_dims:             42
decoder_dims:             21
multiscale_decoder:       False

[model.original]
lowmem:                   True

[model.realface]
input_size:               64
output_size:              128
dense_nodes:              1536
complexity_encoder:       128
complexity_decoder:       512

[model.unbalanced]
input_size:               128
lowmem:                   True
clipnorm:                 False
nodes:                    1024
complexity_encoder:       128
complexity_decoder_a:     384
complexity_decoder_b:     512

[model.villain]
lowmem:                   False

[trainer.original]
preview_images:           14
zoom_amount:              5
rotation_range:           10
shift_range:              5
flip_chance:              50
color_lightness:          30
color_ab:                 8
color_clahe_chance:       50
color_clahe_max_size:     4

```

**Comment by jpchow26** [comment]
appears to be fixed at c2e54bb. graph popups working again.

**Comment by bryanlyon** [comment]
Closing as fix has solved the bug.
